<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CellReel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_sino">
       <attribute name="title">
        <string>&amp;Sinogram</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="SinoWidget" name="widget_sino" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_reco">
       <attribute name="title">
        <string>&amp;Reconstruction</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="RecoWidget" name="widget_reco" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_info">
       <attribute name="title">
        <string>&amp;Information</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="InfoWidget" name="widget_info" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Session"/>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Session">
   <property name="text">
    <string>&amp;New Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="text">
    <string>&amp;Open Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SinoWidget</class>
   <extends>QWidget</extends>
   <header>cellreel.tab_sino</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecoWidget</class>
   <extends>QWidget</extends>
   <header>cellreel.tab_reco</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoWidget</class>
   <extends>QWidget</extends>
   <header>cellreel.tab_info</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
